heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -161,6 +162,19 @@ void killTopologyHandler(String topologyName) throws TopologyRuntimeManagementEx
     LOG.fine("Scheduler killed topology successfully.");
   }
 
+  boolean confirmWithUser(int oldContainerCount, int newContainerCount) {
+    LOG.info(String.format("The present aurora job has %d containers. "
```

Comment:
This code is in scheduler core - you cannot assume that it is aurora specific. It is used by other schedulers. Please change this

Reply:
will remove `aurora`